posthog/models/feature_flag/feature_flag.py

```diff
@@ -47,6 +47,8 @@ class Meta:
         related_name="analytics_dashboards",
         related_query_name="analytics_dashboard",
     )
+    # whether a feature is sending us rich analytics, like views & interactions.
+    has_enriched_analytics: models.BooleanField = models.BooleanField(default=False, null=True, blank=True)
```

Comment:
Should we be concerned that we keep adding fields to our feature flag model tht are more like analytics for the feature flag rather than core to how the feature flag works (like the dashboard fields too above this) ?

Reply:
Not an issue, as long as it's a property of the flag, can go here. The serializers for how a flag works are separate and don't get bloated by anything that exists on the model.